### PR TITLE
Mailer: Change Anthonette to Liz on application emails

### PR DESCRIPTION
**Pull request by islemaster**
[PLC-327](https://codedotorg.atlassian.net/browse/PLC-327) Change "Anthonette Peña" to "Liz Gauthier" on teacher application emails.

### CODE REVIEW

File: dashboard/app/mailers/pd/application/teacher1920_application_mailer.rb [hunk 1]
```diff
@@ -68,7 +68,7 @@ def principal_approval_completed_partner(teacher_application)
       raise "Partner contact email is required, application id #{@application.id}" unless partner_contact_email
 
       mail(
-        from: 'Anthonette Peña <teacher@code.org>',
+        from: 'Liz Gauthier <teacher@code.org>',
```
Comment (islemaster) [inline dashboard/app/mailers/pd/application/teacher1920_application_mailer.rb:71]: Adjusts the "from" header on three emails:

- `principal_approval_completed_partner__with_partner`
- `principal_approval_completed_partner__with_partner_no_contact`
- `principal_approval_completed_partner__without_partner`

File: dashboard/app/views/pd/application/teacher1920_application_mailer/confirmation.html.haml [hunk 2]
```diff
@@ -44,7 +44,7 @@
   Thanks again for applying!
 
 %p
-  Anthonette Peña
+  Liz Gauthier
```
Comment (islemaster) [inline dashboard/app/views/pd/application/teacher1920_application_mailer/confirmation.html.haml:47]: Adjusts the signature on three emails:

- `confirmation__with_partner`
- `confirmation__with_partner_no_contact`
- `confirmation__without_partner`

File: dashboard/app/views/pd/application/teacher1920_application_mailer/principal_approval_completed_partner.html.haml [hunk 3]
```diff
@@ -14,7 +14,7 @@
   Best,
 
 %p
-  Anthonette Peña
+  Liz Gauthier
```
Comment (islemaster) [inline dashboard/app/views/pd/application/teacher1920_application_mailer/principal_approval_completed_partner.html.haml:17]: Adjusts the "from" header on three emails:

- `principal_approval_completed_partner__with_partner`
- `principal_approval_completed_partner__with_partner_no_contact`
- `principal_approval_completed_partner__without_partner`

(Same ones we adjusted the "from" header for, above.)

File: dashboard/app/views/pd/application/teacher1920_application_mailer/_partner_signature.html.haml [hunk 4]
```diff
@@ -8,7 +8,7 @@
   - else
     = link_to @application.regional_partner.name, CDO.studio_url('/pd/regional_partner_contact/new', CDO.default_scheme)
 - else
-  Anthonette Peña
+  Liz Gauthier
```
Comment (islemaster) [inline dashboard/app/views/pd/application/teacher1920_application_mailer/_partner_signature.html.haml:11]: This change to a signature partial changes the signature on many emails:

- `accepted_no_cost_registration__without_partner`
- `confirmation__without_partner`
- `declined__without_partner`
- `principal_approval__without_partner`
- `principal_approval_completed__without_partner`
- `principal_approval_teacher_reminder__without_partner`
- `registration_sent__without_partner`
- `waitlisted__without_partner`

**Review by agealy — APPROVED** [review]

**Review by breville — APPROVED** [review]

**Pull request merged**